lib/anubis.ex

```diff
@@ -1,50 +1,121 @@
 defmodule Anubis do
+  @moduledoc """
+  Anubis is a library for creating command line applications in Elixir.
 
+  Some of the features of Anubis include:
+    - Simple command creation
+    - Simple option definition
+    - Help out of the box
+    - Runtime configuration
+
+  ### Examples:
+
+    ```elixir
+    defmodule AnubisExample do
+      use Anubis
+
+      banner "This is an example application using Anubis."
+
+      rc_file %{
+        username: "user",
+        password: "p4$$w0rd",
+        people: 10
+      }
+
+      option "verbose", :boolean, "Whether or not to be verbose with the output."
+
+      command :fetch, "Fetch something from the internet", AnubisExample.Function.fetch
+    end
+    ```
+  """
+
+  @doc """
+  Sets the banner as the description for the cli application.
```

Comment:
CLI is an abbreviation and should be written in uppercase IMO (no pun intended).